### PR TITLE
Implemented logging of telnet data.

### PR DESCRIPTION
**Pull request by iamxlr8**

### CODE REVIEW

File: heralding/capabilities/telnet.py [hunk 1]
```diff
@@ -64,6 +66,17 @@ def __init__(self, reader, writer, session, loop):
...
+    def get_auxiliary_info(self):
+        data_fields = TelnetWrapper.get_aux_fields()
+        data_reader = {f: self.reader.get_extra_info(f) for f in data_fields}
+        data_writer = {f: self.writer.get_extra_info(f) for f in data_fields}
+        data = {'data_reader': data_reader, 'data_writer': data_writer}
+        return data
+
+    @staticmethod
+    def get_aux_fields():
+        return ['status']
```
[the remark this answers is not in the document]
Comment (iamxlr8) [reply heralding/capabilities/telnet.py]: I read the rfc but could not find relevant information to log.
In rfc 859 i got telnet status option.
It allows the user/process to verify the status of telnet options

[the remark this answers is not in the document]
Comment (iamxlr8) [reply heralding/capabilities/telnet.py]: Please guide if i understood or did anything wrong.

[the remark this answers is not in the document]
Comment (johnnykv) [reply heralding/capabilities/telnet.py]: Did you investigate the actual content of [‘status’]?

[the remark this answers is not in the document]
Comment (iamxlr8) [reply heralding/capabilities/telnet.py]: I could not ..because aux_telnet.log could not be generated.

[the remark this answers is not in the document]
Comment (iamxlr8) [reply heralding/capabilities/telnet.py]: I tried ssh connection too(just for checking).. but again the aux_ssh.log was also not generated.

[the remark this answers is not in the document]
Comment (johnnykv) [reply heralding/capabilities/telnet.py]: Ok, so no one really knows what this code does 😀. I would suggest reopening the pr when you have actual running code and have it logging at least something...

**Comment by johnnykv** [comment]
See my earlier comment. I would also suggest investigating if we can log terminal setting from the client. Try looking for NVT in the RFC.

File: heralding/capabilities/telnet.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (iamxlr8) [reply heralding/capabilities/telnet.py]: I read about 'status'.. so i mentioned it in the get_aux_fields().. and pushed it so that i could get some help.

**Comment by iamxlr8** [comment]
Ok I will try once again. Thanks for the guidance. 😃

**Pull request closed without merging**